<launch>
    <node pkg="odml_visual_odometry" type="visual_odometry_node" name="visual_odometry_node" output="screen">
        <param name="is_classic" type="bool" value="false" />
        <param name="detector_type" type="string" value="SuperPoint" />
        <param name="descriptor_type" type="string" value="SuperPoint" />
        <param name="matcher_type" type="string" value="BF" />
        <param name="selector_type" type="string" value="KNN" />
        <param name="image_height" type="int" value="360" />
        <!-- <param name="image_height" type="int" value="240" /> -->
        <!-- <param name="image_height" type="int" value="120" /> -->
        <param name="image_width" type="int" value="1176" />
        <!-- <param name="image_width" type="int" value="784" /> -->
        <!-- <param name="image_width" type="int" value="392" /> -->
        <!-- <param name="model_name_prefix" type="string" value="superpoint_pretrained" /> -->
        <param name="model_name_prefix" type="string" value="sp_sparse" />
        <param name="model_batch_size" type="int" value="1" />
        <param name="machine_name" type="string" value="laptop" />
        <param name="trt_precision" type="string" value="FP32" />
        <param name="conf_thresh" type="double" value="0.015" />
        <param name="dist_thresh" type="int" value="4" />
        <param name="num_threads" type="int" value="6" />
        <param name="border_remove" type="int" value="4" />
        <param name="stereo_threshold" type="double" value="2.0" />
        <param name="min_disparity" type="double" value="0.25" />
        <param name="refinement_degree" type="int" value="4" />
    </node>
</launch>